microk8s-resources/actions/disable.openebs.sh

```diff
@@ -5,63 +5,116 @@ set -e
 source $SNAP/actions/common/utils.sh
 HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
 KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+OPENEBS_NS="openebs"
```

Comment:
An high-level comment. Is there a way to preempt the disable process if there are volumes or pools configured.

Reply:
@kmova -- There is a check at the end
```bash
read -p "Have you deleted all the pods and PersistentVolumeClaims using OpenEBS PVC? (Y/N): " confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || exit 1
```
This will ask the user if all OpenEBS volumes are deleted. If there is an 'N' in the response, the disable_openebs function is not called.
disable_openebs is called directly without any chance for preemption if the user executes
```bash
microk8s disable openebs:force
```
instead of 
```bash
microk8s disable openebs
```